src/components/ObjectInfo.js

```diff
@@ -0,0 +1,35 @@
+function ObjectInfo({arrangement, onItemPress, side, style}) {
+
+    function onPress({idbtn, gender:gender }) {
+        onItemPress({side, id:idbtn, gender:gender});
+    }
+
+    return (
+        <li className={'UserInfo'+side} style={style}>
```

Comment:
`'UserInfo'+side` works but a bad way to write
please use 
`<li className={`UserInfo${side}`}`